docs/maintainers/adr/0013-removing-deprecated-helpers.md

```diff
@@ -0,0 +1,37 @@
+# play-frontend-hmrc mirrors hmrc-frontend
+
+* Status: accepted
+* Date: 2020-06-24
```

Comment:
Date needs updating.

Reply:
The date is based on the comment in Jira documenting the decision when we made it.